src/nora/components/Tag/Tag.module.scss

```diff
@@ -0,0 +1,23 @@
+@import '../Attributes.css';
+
+.Tag {
+  border-radius: var(--Radius-3);
+  display: inline;
+  align-items: center;
+  padding: 5px;
```

Comment:
It would be preferable to use `var(Space-<N>)` here I think we have 4px and 8px for that. I'm not sure if that's doable. If not, I'd 1) double check with Tom if it has to be this way (if this is just to match the design) 2) I suppose we could revisit this and just go with the hard coded 5px.

Reply:
4 looks a little tight but I think it's okay for now! I'll stick to using the defaults and when we go through a fine-tune everything we can decide when to make exceptions.